spec/models/news_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe(News, type: :model) do
+  let(:news) { create(:news) }
+
+  context 'when title and description fields are empty' do
```

Comment:
you have added https://github.com/thoughtbot/shoulda-matchers so please rewrite all tests as we agreed